addons/pos_restaurant/static/tests/tours/pos_restaurant.js

```diff
@@ -139,6 +139,8 @@ registry.category("web_tour.tours").add("pos_restaurant_sync_second_login", {
             ProductScreen.clickDisplayedProduct("Coca-Cola"),
             ProductScreen.totalAmountIs("2.20"),
             FloorScreen.backToFloor(),
+            FloorScreen.clickTable("4"),
+            FloorScreen.backToFloor(),
```

Comment:
I think something's wrong if you had to get out of the table and go back to it one more time just to see the "order count" in the floor view.